hub-templates/base-hub/Chart.yaml

```diff
@@ -3,3 +3,8 @@ appVersion: '1.0'
 description: Deployment Chart for JupyterHub
 name: base-hub
 version: v0.0.1
+dependencies:
+ - name: jupyterhub
+   # REMEMBER TO CHANGE BASE IMAGE OF images/hub/ WHEN CHANGING THIS
+   version: 0.9.0-n287.hd5f1f881
+   repository: https://jupyterhub.github.io/helm-chart/
```

Comment:
This transition will require a `apiVersion: v2` in the top of the Chart.yaml file I think. That change will also require Helm3, but Helm2 is end of life etc so go for it is my sentiment!